plugins/dex/list/handler.go

```diff
@@ -24,6 +24,9 @@ func NewHandler(keeper *order.DexKeeper, tokenMapper tokens.Mapper, govKeeper go
 		case types.ListMsg:
 			return handleList(ctx, keeper, tokenMapper, govKeeper, msg)
```

Comment:
Why don't we disable `types.ListMsg` as well?

Reply:
`ListMsg` is only effective when the list proposal is approved. I assume there will be no side effects for this after the BEP.
`ListMiniMsg` can be sent by anyone and executed successfully.

Reply:
To me, disable `ListMsg` will make it more symmetric, dismiss the question like what will happen if my listing proposal passed by not executed?

Reply:
To me, disable `ListMsg` will make it more symmetric, it can dismiss the question like what will happen if my listing proposal passed by not executed.

Reply:
Make sense. I'll add it.